source/kraken/data_manager.h

```diff
@@ -137,6 +137,8 @@ class DataManager {
 
         // Build Raptor Data
         data->build_raptor(raptor_cache_size);
+        // Build proximity list NN index
+        data->build_proximity_list();
```

Comment:
You couldn't serialize it?

Reply:
I tried... like really hard :D . I thought flann was using the boost serializer. It turns out that it uses a home made serializer....